apps/studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.tsx

```diff
@@ -189,17 +191,32 @@ export const NavContent = () => {
 
   const UserAccountButton = (
     <Button
+      block
       type="text"
       size="tiny"
       className={cn(
         'mt-3 h-10 [&>span]:relative [&>span]:flex [&>span]:w-full [&>span]:h-full p-0'
       )}
-      block
     >
       <div className="relative w-full h-full flex items-center justify-center">
-        <figure className="absolute left-1.5 min-h-6 min-w-6 bg-foreground rounded-full flex items-center justify-center">
-          <User size={ICON_SIZE - 2} strokeWidth={ICON_STROKE_WIDTH} className="text-background" />
-        </figure>
+        {!!profile?.profileImageUrl && !hasInvalidImg ? (
+          <Image
+            alt={profile.username}
+            src={profile.profileImageUrl}
+            width="24"
+            height="24"
+            className="absolute left-1.5 bg-foreground rounded-full w-6 h-6"
+            onError={() => setHasInvalidImg(true)}
+          />
+        ) : (
+          <figure className="absolute left-1.5 min-h-6 min-w-6 bg-foreground rounded-full flex items-center justify-center">
+            <User
+              size={ICON_SIZE - 2}
+              strokeWidth={ICON_STROKE_WIDTH}
+              className="text-background"
+            />
+          </figure>
+        )}
```

Comment:
I would argue this should be it's own component.
With forwardRef and props all hooked up so we can reuse it